test/password.spec.js

```diff
@@ -187,6 +184,49 @@ describe('$.fn.password', () => {
         })
         .val('_~%8::%nqy^7e~!!z!;N')
         .trigger('keyup')
-    })
+    });
+
+    it('uses no color background image by default', () => {
+      $('#password').password();
+
+      expect('').toEqual($('.pass-colorbar').css('background-image'));
+    });
+
+    it('makes background image style adjustments if turned on', (done) => {
```

Comment:
These `done`s might have been left by me when I merged in and didn't realized we switched to jest. Can we get rid of them?